Sources/TuistGenerator/Linter/SchemeLinter.swift

```diff
@@ -34,6 +34,16 @@ private extension SchemeLinter {
                                                    actionDescription: "the scheme's run action")
                 )
             }
+
+            if let storeKitPath = runAction.options.storeKitConfigurationPath,
+                !FileHandler.shared.exists(storeKitPath)
+            {
+                issues.append(
+                    LintingIssue(
+                        reason: "StoreKit configuration file not found at path \(storeKitPath.pathString)",
+                        severity: .warning
```

Comment:
I'd mark this one as an error. As a user, if I specify the file intentionally, it's because I want it to be part of the project, otherwise, I wouldn't set it at all.